impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java

```diff
@@ -470,14 +475,24 @@ private Response execute(Request request) throws ResourceException {
         Map<String, Object> out = null;
 
         if (response.hasBody()) {
-            out = mapMarshaller.unmarshal(response.getBody(), response.getHeaders().getLinkMap());
+            if ("json".equals(response.getHeaders().getContentType().getSubtype())) {
```

Comment:
This probably isn't generic enough, i.e. this could be `application/ion+json`

Reply:
Of course

Reply:
https://trac.tools.ietf.org/html/draft-ietf-appsawg-media-type-suffix-regs-02#section-3.1

Reply:
(i bring up ion, because Okta is using it for some early access endpoints)